### PR TITLE
Restructure and add test

### PR DESCRIPTION
**Pull request by mcarans**
Note that to run as you do and output downloaded files to data folder, pass parameters --folder=data and ---keep.

### CODE REVIEW

File: run.py [hunk 1]
```diff
@@ -10,7 +12,7 @@
 from hdx.hdx_configuration import Configuration
 from hdx.location.country import Country
 from hdx.utilities.easy_logging import setup_logging
-from hdx.utilities.path import script_dir_plus_file
+from hdx.utilities.path import script_dir_plus_file, temp_dir, get_temp_dir
```
Comment (turnerm) [inline run.py]: I think a couple of these imports are not used

File: tests/fixtures/covid-19 historical cases by country.csv.CSV [hunk 2]
```diff
@@ -0,0 +1,4655 @@
+OBJECTID,ADM0_NAME,DateOfDataEntry,cum_conf,NewCase,CENTER_LON,CENTER_LAT,ADM0_VIZ_NAME,GUID,Short_Name_ZH,Short_Name_FR,Short_Name_ES,Short_Name_RU,Short_Name_AR
```
Comment (turnerm) [inline tests/fixtures/covid-19 historical cases by country.csv.CSV:1]: Are we sure that it's okay to have this data publicly on github? Of course it's already public on HDX so I assume it's fine, but here it's a bit without context, so I just wanted to check.

**Review by turnerm — APPROVED** [review]

File: run.py (hunk 1 above)
Comment (mcarans) [reply run.py]: Thx, fixed

File: tests/fixtures/covid-19 historical cases by country.csv.CSV (hunk 2 above)
Comment (mcarans) [reply tests/fixtures/covid-19 historical cases by country.csv.CSV:1]: I think it should be ok - it is under tests/fixtures, so I think people will realise it is test data

**Pull request merged**